content/bibs/pubs.bib

```diff
@@ -2645,9 +2645,27 @@ @article{Yuste2011
 
 
 
+@inproceedings{jong_naisys_2022,
+  author = {Shin, Jong M and Isik, Leyla and Vogelstein, Joshua T},
+  author+an = {1=trainee; 3=highlight},
+  address = {From Neuroscience to Artificially Intelligent Systems (NAISys), Cold Spring Harbor Laboratory, NY, USA},
+  title = {Measure of human-likelihood in tree-based ensemble model and artificial neural networks},
+  month = {March},
+  year = {2022},
+  url = {https://github.com/jshinm/inductive-bias-experiment/blob/main/posters/NAISys2022-poster.pdf},
+  keywords = {abspos}
+}
 
-
-
+@inproceedings{pedigo_naisys_2022,
+  author = {Pedigo, Benjamin D and Powell, Mike and Bridgeford, Eric W and Winding, Michael and Priebe, Carey E and Vogelstein, Joshua T},
+  author+an = {1=trainee; 2=trainee; 3=trainee; 6=highlight},
```

Comment:
I changed back to its original form b/c this reflects on Jovo's CV, so the highlight is referring to Jovo